pkg/kubelet/networks.go

```diff
@@ -28,14 +26,14 @@ type networkHost struct {
 	kubelet *Kubelet
```

Comment:
I don't think we need `networkHost` anymore. kubelet now implements the `Host` interface directly. We can just move those methods to this file.